x-pack/test_serverless/shared/config.base.ts

```diff
@@ -51,6 +51,8 @@ export default async () => {
       from: 'serverless',
       files: [idpPath, jwksPath],
       serverArgs: [
+        'http.host=0.0.0.0',
```

Comment:
We don't want to accidentally open up elasticsearch to the internet on our CI agents.  Can we find a way to make this work with `localhost`?

Reply:
@gergoabraham I think its ok to change to `127.0.0.1`

Reply:
@elastic/appex-qa strongly suggests not to add any server arguments for both Kibana & ES in shared FTR config:
If your tests rely on this argument for Kibana to be started with, your tests will work locally and Kibana CI, but won't work while running against MKI since server args defined in config file are completely ignored while testing the real serverless project. 
We understand that some tests require custom arguments (e.g. feature flags), but it should be added in the separate FTR config that won't be run against MKI.
Is it required by all Cypress tests?

Reply:
@dmlemeshko - I don't think its required. I remember us having an issue a while back with ES (maybe when running local CLI scripts) and this seemed to have fixed it, but I don't recall what that was now.

I'm ok removing it, unless @gergoabraham hit a problem that requires it. if thats the case, then I would move this change to the test specific suite rather than to make the change here to the `base` config.

Reply:
there was an issue that ES wasn't available locally,, and this server arg helped. I'm not sure what happened in the background, but now it works, so I was able to remove it easily: 4a0cea842f8926aa77af0495d68c3af1313c856f

thanks all for the feedback!